webpack.config.ts

```diff
@@ -11,28 +11,63 @@ import CopyWebpackPlugin from 'copy-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 import webpack, {Configuration} from 'webpack';
 
-function parseFeatureDetails(id: FeatureID): FeatureMeta {
-	const content = readFileSync(`source/features/${id}.tsx`, {encoding: 'utf-8'});
-	const fields = ['disabled', 'description', 'screenshot'] as const;
+const readmeContent = readFileSync('readme.md', 'utf-8');
+
+function stripLinks(markdownText: string): Partial<FeatureMeta> {
+	const urls: string[] = [];
+	const description = markdownText.replace(/\[(.+?)]\((.+?)\)/g, (_match, title, url) => {
+		urls.push(url);
+		return title;
+	});
+
+	return {
+		description,
+		screenshot: urls.find(url => url.startsWith('https://user-images.githubusercontent.com/')) ?? urls[0]
+	};
+}
 
+function parseFeatureDetails(id: FeatureID): FeatureMeta {
 	const feature: Partial<FeatureMeta> = {id};
-	for (const field of fields) {
-		const value = new RegExp(`\n\t${field}: '([^\\n]+)'`).exec(content)?.[1];
-		if (value) {
-			const validValue = value.trim().replace(/\\'/g, '’'); // Catch trailing spaces and incorrect apostrophes
-			if (value !== validValue) {
-				throw new Error(stripIndent(`
-					❌ Invalid characters found in \`${id}\`. Apply this patch:
-
-					- ${field}: '${value}'
-					+ ${field}: '${validValue}'
-				`));
-			}
 
-			feature[field] = value.replace(/\\\\/g, '\\');
+	const lineRegex = new RegExp(`^- \\[\\]\\(# "${id}"\\)(?: 🔥)? (.+)$`, 'm');
+	const lineMatch = lineRegex.exec(readmeContent);
+	if (lineMatch) {
+		Object.assign(feature, stripLinks(lineMatch[1]));
```

Comment:
Just inline `stripLinks`:

```suggestion
 	const urls: string[] = [];
 	feature.description = markdownText.replace(/\[(.+?)]\((.+?)\)/g, (_match, title, url) => {
 		urls.push(url);
 		return title;
 	});
	feature.screenshot = urls.find(url => /\.(png|gif)$/.test(url));
```